test/lint/blocking_commands.rb

```diff
@@ -1,5 +1,8 @@
 module Lint
   module BlockingCommands
+    TIMEOUT_SEC = 0.01
```

Comment:
I'm not a huge fan of `TIMEOUT_SEC`, as it's not very meaningful. What about `LOW_TIMEOUT` ?